flux/grids.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+
+def setgrid(nz,zfac,zmax):
```

Comment:
Would be good to move this into flux.thermal since it's only used for conductionQ. (And then delete the grids.py file.)

Reply:
I just see flux.thermal.pyx ... do you mean there? Not sure if it works in the pyx file.

Reply:
Yeah, you can just drop it into thermal.pyx. No need to do anything special